infrastructure/docker/README.md

```diff
@@ -49,3 +50,24 @@ docker build -f ./infrastructure/docker/Dockerfile-notebook -t <IMAGE_NAME> .
 
 ### Versions
 - Python == 3.7
+
+## Repository Server
+
+An image of the repository server.
+
+### Build
+To build image just run from the root of the project:
+
+```shell
+make build-repository-server
+```
+
+or in case you want to customize as much as possible your build:
+
+```shell
+docker build -f ./infrastructure/docker/Dockerfile-repository-server -t <IMAGE_NAME> .
+```
+
+### Versions
+- Python == 3.9
```

Comment:
Ray and Jupyter are using Python 3.7 as per this doc... I think 3.9 is the default now (cc @Tansito @IceKhan13 to confirm, if that's true I can open a separate PR to bump those versions)

Reply:
confirming :) 3.9 if default now everywhere since https://github.com/Qiskit-Extensions/quantum-serverless/pull/198

Reply:
Looks like it's just the readme that needs updating (dockerfiles were already done)... opened #240 to update readme